src/main/java/core/basesyntax/MainApp.java

```diff
@@ -12,5 +12,17 @@
  * - Отправить все машини на работу и остановить работу всех машин.
  */
 public class MainApp {
+    public static void main(String[] args) {
+        Machine track = new Track();
+        Machine bulldozer = new Bulldozer();
+        Machine excavator = new Excavator();
+        startMachine(track);
+        startMachine(bulldozer);
+        startMachine(excavator);
+    }
 
+    private static void startMachine(Machine machine) {
```

Comment:
first of all the name of this method should have been startAndStopMachine(smth like that) and secondly let's remove it )